package.json

```diff
@@ -10,6 +10,8 @@
   "license": "MIT",
   "main": "source/index.js",
   "scripts": {
+    "build": "./node_modules/.bin/gulp build",
```

Comment:
This could be just `"build": "gulp build"`, as Node adds `./node_modules/.bin/` to `PATH` on its own.